keyboards/keebio/sinc/rev3/keymaps/vial/vial.json

```
@@ -0,0 +1,375 @@
+{
+  "name": "Sinc Rev. 3",
+  "vendorId": "0xCB10",
+  "productId": "0x3267",
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_rgb_matrix"],
```

```suggestion
  "lighting": "vialrgb",
```
VIA ≠ Vial. Please reference [the Vial lighting documentation](https://get.vial.today/docs/lighting.html#rgb-matrix--vialrgb).

---

Done.